### PR TITLE
PS - Intro

### PR DESCRIPTION
**Pull request by genintho**
- [ ] Rename file

### CODE REVIEW

File: _posts/2016-10-10-paperspider-week1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+date: 2016-10-10 19:00:00
+---
+
+# PaperSpider weekly status #1
+
+
+PaperSpider is the idea of a project I had in th back of my mind for a very long time. Every time you need to do a task involving a lots of different documents, it is just a giant pain. Some document are on paper, some are in your email account(s?), others are behind some accounts, on some folder, etc.
+It is a disaster, you could not believe.
+
+One obvious solution is to copy stuff on your laptop and organized the document in some kind of folder system/naming convention system. It works. But I am lazy, and I would like something a little bit more automated, importing, classifying, indexing and archiving the documents for me with very little interaction from my part.
```
Comment (Sissik) [inline _posts/2016-10-10-paperspider-week1.md]: document in some kind of folder -> document in a folder

Comment (Sissik) [inline _posts/2016-10-10-paperspider-week1.md]: something a little bit more automated -> something a bit more automated

File: _posts/2016-10-10-paperspider-week1.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+That's for the MVP. Many additional extensions can be imagined to simplify our digital life from sharing document, with a lawyer for example, to documents checklist helping application to different things.
```
Comment (Sissik) [inline _posts/2016-10-10-paperspider-week1.md]: Many additional extensions -> Many more additional extensions

File: _posts/2016-10-10-paperspider-week1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+I kept rumbling about this first world problem with my friends and to myself with each new tax season or visa application. One day, Mike reached a tipping point where he saw himself be helped by such a tools. And that was it. We were going to build PaperSpider.
+
+We are perfectly aware that this tool will be useful to a minority. But we are seeing this as an opportinity to learn and keep working together. We want to build it like it is 2005, trying to experiment with the UI/UX and be opinated, even if it will not please everybody.
```
Comment (Sissik) [inline _posts/2016-10-10-paperspider-week1.md]: and be opinated -> (what does it mean?) and be opinionated

File: _posts/2016-10-10-paperspider-week1.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+
+PaperSpider is the idea of a project I had in th back of my mind for a very long time. Every time you need to do a task involving a lots of different documents, it is just a giant pain. Some document are on paper, some are in your email account(s?), others are behind some accounts, on some folder, etc.
+It is a disaster, you could not believe.
+
+One obvious solution is to copy stuff on your laptop and organized the document in some kind of folder system/naming convention system. It works. But I am lazy, and I would like something a little bit more automated, importing, classifying, indexing and archiving the documents for me with very little interaction from my part.
+
+That's for the MVP. Many additional extensions can be imagined to simplify our digital life from sharing document, with a lawyer for example, to documents checklist helping application to different things.
+
+I kept rumbling about this first world problem with my friends and to myself with each new tax season or visa application. One day, Mike reached a tipping point where he saw himself be helped by such a tools. And that was it. We were going to build PaperSpider.
```
Comment (Sissik) [inline _posts/2016-10-10-paperspider-week1.md]: **rumbling** (a continuous deep, resonant sound like distant thunder.
"the steady rumble of traffic") -> **grumbling** (complain or protest about something in a bad-tempered but typically muted way.)

File: _posts/2016-10-10-paperspider-week1.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "PaperSpider - Introduction"
+date: 2016-10-10 19:00:00
+---
+
+# PaperSpider weekly status #1
+
+
+PaperSpider is the idea of a project I had in th back of my mind for a very long time. Every time you need to do a task involving a lots of different documents, it is just a giant pain. Some document are on paper, some are in your email account(s?), others are behind some accounts, on some folder, etc.
```
Comment (michaelkitson) [inline _posts/2016-10-10-paperspider-week1.md]: `in th back` => `in the back`, `a lots` => `a lot`, `Some document` => `Some documents`

**Pull request merged**